ThunderCloud/AccessibleCollectionView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AccessibleCollectionView.swift
+//  First Aid 2.0
+//
+//  Created by Simon Mitchell on 04/09/2019.
+//  Copyright © 2019 3 SIDED CUBE. All rights reserved.
+//
+
+import UIKit
+
+/// A subclass of `UICollectionViewCell` that can be used in `UITableViewCell`
+/// and returns the correct value from `accessibilityElementCount()`
+public class AccessibleCollectionView: UICollectionView {
+
+    override open func accessibilityElementCount() -> Int {
+        
+        guard let dataSource = dataSource else {
+            return 0
+        }
+        
+        let numberOfSections = dataSource.numberOfSections?(in: self) ?? 1
```

Comment:
Why is this defaulting to 1 if there's no sections? :)

Reply:
Not sure... may have been a copy pasta ;) will adjust!